packages/ds-sam-sdk/src/auction.ts

```diff
@@ -14,6 +14,24 @@ const logValidators = (validators: AuctionValidator[]) => {
 
 export const EPSILON = 1e-4
 
+type ReputationValues = {
+  spendRobustReputation: number
+  adjSpendRobustReputation: number
+  adjMaxSpendRobustDelegation: number
+}
+
+const setReputation = (validator: AuctionValidator, values: ReputationValues): ReputationValues => {
+    const oldValues = {
+      spendRobustReputation: validator.values.spendRobustReputation,
+      adjSpendRobustReputation: validator.values.adjSpendRobustReputation,
+      adjMaxSpendRobustDelegation: validator.values.adjMaxSpendRobustDelegation,
+    }
+    validator.values.spendRobustReputation = validator.values.spendRobustReputation
+    validator.values.adjSpendRobustReputation = validator.values.adjSpendRobustReputation
+    validator.values.adjMaxSpendRobustDelegation = validator.values.adjMaxSpendRobustDelegation
```

Comment:
```suggestion
    validator.values.spendRobustReputation = values.spendRobustReputation
    validator.values.adjSpendRobustReputation = values.adjSpendRobustReputation
    validator.values.adjMaxSpendRobustDelegation = values.adjMaxSpendRobustDelegation
```
This would do no update, I assume the intention was to store the `values` from the param.